pom.xml

```diff
@@ -391,10 +396,13 @@
         </snapshotRepository>
     </distributionManagement>
     <properties>
-        <carbon.identity.version>5.12.153</carbon.identity.version>
+        <carbon.identity.version>5.15.47</carbon.identity.version>
```

Comment:
Shall we add a comment saying why this update is done and why the import version range is not changed ?

Reply:
Added a comment https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/169#issuecomment-1434065115